client/my-sites/subscribers/components/subscribers-header/subscribers-header.tsx

```diff
@@ -1,36 +1,58 @@
-import { Button, Gridicon } from '@automattic/components';
-import { HelpCenter } from '@automattic/data-stores';
+import { Gridicon } from '@automattic/components';
+import { HelpCenter, updateLaunchpadSettings } from '@automattic/data-stores';
 import { useLocalizeUrl } from '@automattic/i18n-utils';
+import { useQueryClient } from '@tanstack/react-query';
+import { Button } from '@wordpress/components';
```

Comment:
The automattic version seems to have been deprecated so I switched this to the WordPress core version